File: Mobile_development/MyApplication/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private ImageButton startButton;
    private ImageView dogImage;
    private TextView helloText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        startButton = findViewById(R.id.startButton);
        dogImage = findViewById(R.id.dogImage);
        helloText = findViewById(R.id.helloText);
    }

    // Метод для кнопки Start
    public void onStartButtonClick(View view) {
        // Скрыть кнопку
        startButton.setVisibility(View.GONE);

        // Показать картинку и текст
        dogImage.setVisibility(View.VISIBLE);
        helloText.setVisibility(View.VISIBLE);
    }
}

File: Mobile_development/task4/app/src/main/java/com/example/scrolling_4/MainActivity.kt
package com.example.scrolling_4


import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val poemTextView = findViewById<TextView>(R.id.poemText)

        val poemText = """
            Был у кошки сын приемный —
Не котенок, а щенок,
Очень милый, очень скромный,
Очень ласковый сынок.
 
Без воды и без мочала
Кошка сына умывала;
Вместо губки, вместо мыла
Языком сыночка мыла.
 
Быстро лижет язычок
Шею, спинку и бочок.
Кошка-мать —
Животное
Очень чистоплотное.
 
Но подрос
Сынок приемный,
И теперь он пес
Огромный.
 
Бедной маме не под силу
Мыть лохматого верзилу.
На громадные бока
Не хватает языка.
 
Чтобы вымыть шею сыну,
Надо влезть ему на спину.
— Ох, — вздохнула кошка-мать, —
Трудно сына умывать!
 
Сам плескайся, сам купайся,
Сам без мамы умывайся!
…
Сын купается в реке,
Мама дремлет на песке.
        """.trimIndent()

        poemTextView.text = poemText
    }
}
